ivadomed/utils.py

```diff
@@ -366,29 +365,38 @@ def segment_volume(folder_model, fname_image, fname_roi=None, gpu_number=0):
     with open(fname_model_metadata, "r") as fhandle:
         context = json.load(fhandle)
 
+    # LOADER
+    loader_params = context["loader_parameters"]
+    slice_axis = AXIS_DCT[loader_params['slice_axis']]
+    metadata = {}
+    fname_roi = None
+    if fname_prior is not None:
+        if 'roi_params' in loader_params and loader_params['roi_params']['suffix'] is not None:
+            fname_roi = fname_prior
+        if 'object_detection_params' in context and \
+                context['object_detection_params']['object_detection_path'] is not None:
+            imed_obj_detect.bounding_box_prior(fname_prior, metadata, slice_axis)
+
     # TRANSFORMATIONS
     # If ROI is not provided then force center cropping
     if fname_roi is None and 'ROICrop' in context["transformation"].keys():
         print("\nWARNING: fname_roi has not been specified, then a cropping around the center of the image is performed"
               " instead of a cropping around a Region of Interest.")
         context["transformation"] = dict((key, value) if key != 'ROICrop'
-                                                    else ('CenterCrop', value)
-                                                    for (key, value) in context["transformation"].items())
+                                         else ('CenterCrop', value)
+                                         for (key, value) in context["transformation"].items())
```

Comment:
I would recommend to move these lines after L375 (ie fname_roi = fname_prior).. so that its kind of grouped. But really minor, you can ignore this comment without any pb.

Reply:
Done :)